public/app/plugins/panel/timeline/timeline.ts

```diff
@@ -73,10 +74,11 @@ export function getConfig(opts: TimelineCoreOptions) {
       let mark = document.createElement('div');
       mark.classList.add('bar-mark');
       mark.style.position = 'absolute';
-      mark.style.background = 'rgba(255,255,255,0.4)';
+      mark.style.background = 'rgba(255,255,255,0.2)';
```

Comment:
I toned down the highlight a bit, it felt a bit on the strong side but happy to change it back